Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/02-ListsOfSyncUps/ListsOfSyncUps.tutorial

```diff
@@ -54,6 +54,9 @@
         @Code(name: "Models.swift", file: ListsOfSyncUps-01-code-0003.swift)
       }
       
+      <!--
+      NB: Removing the Tagged discussion for now because it was too confusing for readers.
+      
```

Comment:
let's just drop the tagged digression. it's causing too much confusion.

Reply:
It helped me to re-discover tagged again. I agree - it can be confusing for junior devs, but it's a pity.